packages/wrangler/src/environment-variables/misc-variables.ts

```diff
@@ -57,3 +57,11 @@ export const getCloudflareApiBaseUrl = getEnvironmentVariableFactory({
 			? "https://api.staging.cloudflare.com/client/v4"
 			: "https://api.cloudflare.com/client/v4",
 });
+
+// Should we sanitize debug logs? By default we do, since debug logs could be added to GitHub issues and shouldn't include sensitive information
+export const getSanitizeLogs = getEnvironmentVariableFactory({
+	variableName: "WRANGLER_SANITIZE_DEBUG_LOGS",
```

Comment:
To match the existing, related env vars `WRANGLER_LOG` and `WRANGLER_LOG_PATH`:
```suggestion
	variableName: "WRANGLER_LOG_SANITIZE",
```